src/commands/punsubscribe.json

```diff
@@ -1,7 +1,7 @@
 {
     "PUNSUBSCRIBE": {
         "summary": "Stops listening to messages published to channels that match one or more patterns.",
-        "complexity": "O(N+M) where N is the number of patterns the client is already subscribed and M is the number of total patterns subscribed in the system (by any client).",
+        "complexity": "O(N) where N is the number of patterns the client is already subscribed.",
```

Comment:
isn't it `the number of patters to unsubscribe`?, i.e. if if runs without arguments it unsubscribes all, and if runs with arguments it unsubscribes just a few (or one)

if that's the case, this change should probably be reflected to the other *UNSUBSCRIBE commands.